src/app/components/Footer/Footer.tsx

```diff
@@ -33,6 +33,7 @@ export default function Footer() {
       <div
         className={classNames(
           styles['footer-phdi'],
+          'footer-phdi',
           'usa-header--basic bg-background-teal',
```

Comment:
I believe we can remove these styles entirely.

Reply:
you are correct